client/src/HeaderMenu.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './HeaderMenu.css'
+import './App.css'
+
+
+class HeaderMenu extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            whoohoo: "but look, at least the page is blue"
```

Comment:
If this is going to be a static string, then you could just hard-code it into the jsx in the return statement below. If you are only using state to keep track of some variables that hold data, you can make a functional component, which is just a component in function form like so:
```js
function HeaderMenu(props) {
  let myVar = 'this is my string'

  return(
    <h3>{myVar}</h3>
  )
}
```
then instead of using your state, you can just use ```myVar```